modules/xz/5.4.5.bcr.3/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/tukaani-project/xz/releases/download/v5.4.5/xz-5.4.5.tar.gz",
+    "integrity": "sha256-E1yQuTSu6PvA1Gfeh6Bctw1ifaNqvlGMNXqHNwnlt9Y=",
+    "strip_prefix": "xz-5.4.5",
+    "patches": {
+        "patch.diff": "sha256-IJv4ilebferiAUe4xqCIKxYMfhbShH4P7v57v55dcp8="
```

Comment:
Since this patch is gigantic, could you split it into two parts, the patch from the previous version and your patch on top of it? Then you could reference the previous patch via `../../<previous version>/patches/patch.diff`, which would greatly simplify the review.

Reply:
that makes sense, did not know was possible.

Reply:
@fmeum done